packages/eslint-plugin/package.json

```diff
@@ -16,11 +16,11 @@
   "dependencies": {
     "@griffel/eslint-plugin": "^1.6.3",
     "@rnx-kit/eslint-plugin": "^0.5.3",
```

Comment:
you need to bump this one as well - we are having ts-eslint v6 in node_modules because of this